style.css

```diff
@@ -15,7 +18,13 @@ input {
     border-radius: 10px;
```

Comment:
marking here,
I checked the frontend and found out the `width` property is used to reverse the effects of padding applied to the same input
This is not the recommended

Take a look at box-sizing property for a better and reliable solution.

Reply:
Even if I remove the width property, and use display:block; the input still doesn't take up the whole width.<img width="1422" alt="Screenshot 2023-06-19 at 7 08 26 PM" src="https://github.com/sahibkaur/tip-calculator/assets/45356642/6ff38899-076c-428b-9bc1-d6b4f47aa5b5">

I searched on the internet and it says, I'll have to specify - width:100%.
https://stackoverflow.com/questions/35059569/html-input-element-not-taking-100-width

Reply:
Because 100% is 100% and 95% varies a lot in different screens